### PR TITLE
fix(docs): Update documentation on how to create a milestone

### PR DESCRIPTION
**Pull request by rfk**
I realized that these are out of date and may have lead to some hiccups in creating milestone for CAD phase 3; this updates them to match the reality of what the script currently does.  @mozilla/fxa-devs r?

### CODE REVIEW

File: scripts/sync_milestones.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ var ghlib = require('./ghlib.js')
 var P = require('bluebird')
 
 var ACTIVE_FEATURE_LABELS = {
+  "designin": true,
```
Comment (philbooth) [inline scripts/sync_milestones.js]: Sorry for the belated drive-by but it looks like `"designing"` is missing the trailing `g` here.

Comment (rfk) [inline scripts/sync_milestones.js]: My bad, I pushed a quick update to this while @vladikoff was reviewing and I think this slipped in during the race; I fixed it in https://github.com/mozilla/fxa/commit/a5f1bebad1c1f95da050ef3554b67b9eb560b14a

Comment (vladikoff) [inline scripts/sync_milestones.js]: 👍
